### PR TITLE
New package: foliate-1.1.0

### PR DESCRIPTION
**Pull request by not-chicken**

### CODE REVIEW

**Comment by not-chicken** [comment]
Updated to version 1.1.0
Please review.

File: srcpkgs/foliate/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'foliate'
+pkgname=foliate
+version=1.1.0
+revision=1
+build_style=meson
+configure_args="--prefix=/usr"
```
Comment (jnbr) [inline srcpkgs/foliate/template]: This is already set by the build-style and can be removed.

Comment (not-chicken) [reply srcpkgs/foliate/template]: Thanks for the review. I removed that.

File: srcpkgs/foliate/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'foliate'
+pkgname=foliate
+version=1.1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gjs-devel glib-devel"
+makedepends="meson ninja"
```
Comment (jnbr) [inline srcpkgs/foliate/template]: makedepends is for the target architecture. If you need them at all, these two should go into hostmakedepends

File: srcpkgs/foliate/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'foliate'
+pkgname=foliate
+version=1.1.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config gjs-devel glib-devel"
```
Comment (jnbr) [inline srcpkgs/foliate/template]: you porbably onlyneed gjs, not gjs-devel

Comment (not-chicken) [reply srcpkgs/foliate/template]: Indeed, thanks.

File: srcpkgs/foliate/template (hunk 2 above)
Comment (not-chicken) [reply srcpkgs/foliate/template]: Done.

**Pull request merged**